agent/handlers/agentapi/taskprotection/v1/handlers/handlers.go

```diff
@@ -150,7 +150,7 @@ func UpdateTaskProtectionHandler(state dockerstate.TaskEngineState,
 func (factory TaskProtectionClientFactory) newTaskProtectionClient(credentialsManager credentials.Manager, task *apitask.Task) (api.ECSTaskProtectionSDK, int, error) {
 	taskRoleCredential, ok := credentialsManager.GetTaskCredentials(task.GetCredentialsID())
 	if !ok {
-		return nil, http.StatusBadRequest, errors.New("invalid Request: no task IAM role credentials available for task")
+		return nil, http.StatusBadRequest, errors.New("Invalid Request: no task IAM role credentials available for task")
```

Comment:
Why are we changing to uppercase? I think errors should be in lowercase?

Reply:
We are debating on what the output looks like. Here I am unifying all messages sent back to user first and changing accordingly later. Before, things like "Failed to decode" and "invalid request" makes it confusing.

Reply:
oh this error is sent to the user. Gotcha.